packages/admin-ui/src/date-picker/date-picker.state.ts

```diff
@@ -13,24 +9,41 @@ import { useDateFieldState } from '../date-field'
 import { toUTCString } from '../calendar/utils'
 // import type { RangeValueMinMax } from '../calendar'
 
-export type DatePickerInitialState = ValueBase<string> &
-  Validation &
-  PickerInitialState &
-  Pick<Partial<DateFieldInitialState>, 'formatOptions' | 'placeholder'> & {
-    /**
-     * Whether the element should receive focus on render.
-     */
-    autoFocus?: boolean
-  }
+export type DatePickerInitialState =
+  ValueBase<string> & {} & PickerInitialState &
+    Pick<Partial<DateFieldInitialState>, 'formatOptions' | 'placeholder'> & {
+      /**
+       * Whether should receive focus on render
+       * @default false
+       */
+      autoFocus?: boolean
+      /**
+       * Whether is invalid
+       * @default false
+       */
+      invalid?: boolean
+      /**
+       * Whether is required
+       * @default false
+       */
+      required?: boolean
+      /**
+       * Whether is disabled
+       * @default false
+       */
+      disabled?: boolean
+    }
 
 // TODO: support min-max values
 export const useDatePickerState = (props: DatePickerInitialState = {}) => {
   const {
     defaultValue = toUTCString(new Date()),
     // minValue,
```

Comment:
Should all those commented lines be removed?

Reply:
Since it's an experimental component and it has the `TODO` message above, I think it's good to leave it here to act as a reminder. What do you think?